src/engine/basics/Settings.hxx

```diff
@@ -177,6 +180,8 @@ public:
 
   using SettingsData::operator=;
 
+  uint32_t getFontSize(const std::string& elmType) const;
```

Comment:
Settings should just have read and write as functions. When adding a new parameter to settings, you will add it to the SettingsData struct.

Reply:
settings data only for data, I think. But workaround code better here

Reply:
why is this resolved?

Reply:
we want use only one font size for all elements?